technic/helpers.lua

```diff
@@ -77,6 +77,8 @@ function technic.get_RE_charge(stack)
 	if wear_factor then
 		local wear = stack:get_wear()
 		return math.floor((wear > 0 and 65536 - wear or 0) / wear_factor + 0.5)
+	else
+		minetest.log("warning", "technic.get_RE_charge item not registered as power tool: "..stack:get_name())
```

Comment:
Might want to make that 'verbose' instead of 'warning' but not sure. During quick testing did not see any warnings logged, it might be useful as a warning to encourage better checks on power tool related mods.
Basically encourage mods to do things only when they know it is appropriate instead of trying and seeing if it fails or not.

Reply:
Logging can be fine tuned later if needed